src/Plugin.php

```diff
@@ -65,6 +65,9 @@ public static function init() {
     // Add teaser image to RSS feeds.
     add_action('rss2_item', __NAMESPACE__ . '\Feed::rss2_item');
 
+    // Fix Facebook embeds to keep within wrapper borders.
+    add_filter('oembed_fetch_url', __CLASS__ . '::oembed_fetch_url', 10, 3);
```

Comment:
1. There's a oEmbed section at the beginning of `init()` – which adds support for Facebook as an oEmbed provider in the first place.  (WordPress Core does not support Facebook natively.)

1. The comment could be more clear – _"Ensure that Facebook embeds do not exceed available content width."_